stories/point.stories.jsx

```diff
@@ -0,0 +1,72 @@
+import Point from '../app/components/point'
+import PointLeadButton from '../app/components/point-lead-button'
+import React from 'react'
+import Theme from '../app/components/theme'
+
+function DemInfoTestComponent(props) {
+  const { isHovered, vState } = props
+  const theme = Theme
+  return (
+    <div
+      style={{
+        color: isHovered || vState === 'mouseDown' ? theme.colors.success : '#5D5D5C',
+        ...theme.font,
+        fontSize: '1rem',
+        fontWeight: '400',
+        lineHeight: '1.5rem',
+      }}
+    >
+      DemInfo | Component
+    </div>
+  )
+}
+
+export default {
+  component: Point,
+  args: {
+    subject: 'Phasellus diam sapien, placerat id sollicitudin eget',
+    description:
+      'Cras porttitor quam eros, vel auctor magna consequat vitae. Donec condimentum ac libero mollis tristique.',
+  },
+}
+
+export const Primary = { args: { vState: 'default', children: <DemInfoTestComponent /> } }
+export const PrimaryMouseDown = { args: { vState: 'mouseDown', children: <DemInfoTestComponent /> } }
```

Comment:
I talked with the designer today. What she meant by mouseDown we would call "selected".  After the user clicks on the the Point, it goes into the selected state.  So we should rename everthing to do with mouseDown to "selected".